spec/features/thredded/visitor_signs_in_spec.rb

```diff
@@ -1,14 +1,19 @@
 # frozen_string_literal: true
 require 'spec_helper'
 
-feature 'Signing in' do
-  scenario 'Visitor with existing account signs in' do
-    create(:user, name: 'joe', email: 'joe@example.com')
-    visit new_user_session_path
-    fill_in 'name', with: 'joe'
-    click_button 'Sign in'
+feature 'Signing in with existing account' do
+  let(:user) { PageObject::User.new(create(:user, name: 'joe', email: 'joe@example.com')) }
 
-    expect(page).to have_content('joe')
-    expect(page).to have_content('Sign out')
+  scenario 'directly' do
+    user.log_in
+    expect(user).to be_signed_in
+  end
+
+  scenario 'after clicking a follow link' do
+    PageObject::Topic.new(create(:topic, with_posts: 1)).visit_topic
+    click_button I18n.t('thredded.topics.follow')
+    expect(page.status_code).to eq 403
+    user.fill_in_sign_in_form_and_submit
+    expect(page).to have_text(I18n.t('thredded.topics.followed_notice'))
```

Comment:
For me this is a little more difficult to follow as we're jumping around from several different methods to test the feature - page object, raw capybara, page object, capybara.

Would you be averse to implementing the test with only page objects?

```ruby
topic.visit_topic
topic.follow_topic
expect(topic).to be_forbidden

user.fill_in_sign_in_form_and_submit
expect(topic).to be_followed
```

Or something to that effect.